src/util/heap/layout/mod.rs

```diff
@@ -3,17 +3,11 @@ pub mod heap_parameters;
 pub mod vm_layout_constants;
 pub mod mmapper;
 pub use self::mmapper::Mmapper;
-#[cfg(target_pointer_width = "32")]
 mod byte_map_mmapper;
-#[cfg(target_pointer_width = "32")]
 pub use self::byte_map_mmapper::ByteMapMmapper;
-#[cfg(target_pointer_width = "64")]
 mod fragmented_mapper;
-#[cfg(target_pointer_width = "64")]
```

Comment:
Does `FragmentedMmapper` work for 32 bits? If not, we do not need to compile it for 32 bits.

Reply:
Both FragmentedMmapper and Map64 are removed from the 32-bit build.